libcxx/test/std/ranges/range.adaptors/range.enumerate/iterator/iter_move.pass.cpp

```diff
@@ -0,0 +1,75 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+
+// <ranges>
+
+// class enumerate_view
+
+// class enumerate_view::iterator
+
+// friend constexpr auto iter_move(const iterator& i)
+//   noexcept(noexcept(ranges::iter_move(i.current_)) &&
+//             is_nothrow_move_constructible_v<range_rvalue_reference_t<Base>>);
```

Comment:
Copying an old comment -- I don't think we're checking the `is_nothrow_move_constructible_v` constraint?